### ISSUE TITLE
[BUG] [Linter] T002 triggers on the wrong line

### ISSUE THREAD

**Issue by malo-cayocca** [Riverside-Healthcare/djLint#405]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info

 - OS: Ubuntu 20.04
 - Python Version 3.8.10
 - djLint Version 1.17.2
 - template language: django

## Issue

1. I get a `T002` false positive when a correctly formatted `include` tag is followed by a line that is also correctly formatted. The order of appearance of said lines seems to matter.

2. Similarly, the `T002` seems to be triggered on the wrong line when there are several `include` tags and one uses single quotes in a `with` statement.

## How To Reproduce

1. The following code will trigger a `T002` error on the first line
```
{% include "template.html" %}
<div {% for name, value in data %} data-{{ name }}='{{ value }}'{% endfor %}/>
```

However, neither of the following will trigger an error
```
{% include "template.html" %}
<div {% for name, value in data %} data-{{ name }}="{{ value }}"{% endfor %}/>
```
```
<div {% for name, value in data %} data-{{ name }}='{{ value }}'{% endfor %}/>
{% include "template.html" %}
```

2. The following code will trigger a `T002` on the first line
```
{% include "template.html" %}
{% include "template.html" %}
{% include "template.html" with type='mono' %}
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]

[tool.djlint]
require_pragma = true
profile = "django"
custom_blocks = "stage, cache, flatblock"
blank_line_after_tag = "extends, endblock, endflatblock"
max_line_length = 120
ignore = "H021,H023"

**Comment by christopherpickering** [comment]
@malo-cayocca  Thanks for the detailed report 👍🏽
